### DISCUSSION TITLE
Transforming your developer experience with GitHub Codespaces [DEV2701D]

### DISCUSSION THREAD

**Discussion by MylesBorins** [community/community#74506] [Codespaces]
**Speakers**: @brianbeck @MylesBorins
Social: You can Brian on [LinkedIn](https://www.linkedin.com/in/bcbeck/). You can follow Myles on [Twitter](https://twitter.com/MylesBorins), [LinkedIn](https://www.linkedin.com/in/mylesborins), or [Web](https://myles.dev/)

> Developers are looking for ways to explore APIs, build, and launch as quickly and effectively as possible. PayPal is using GitHub Codespaces to provide developers the ability to see their APIs in action and modify or enhance their code with a PayPal VS Code plugin, and clone the code to their own repo.

Thank you for joining my/our session/workshop! If you have any questions or feedback regarding what was presented, please feel free to add them in the comments and we'll do our best to answer.

**Comment by Chai-NED** [comment]
I'd love to see a recording if it's available.

**Reply by brianbeck** [reply]
They didn’t record the demos. However, you can read about the integration and see a video overview here:

https://dev.to/paypaldeveloper/paypal-streamlines-checkout-integration-workflows-with-github-codespaces-2j59
